src/main/scala/org/hatdex/hat/api/json/UuidMarshalling.scala

```diff
@@ -12,12 +12,13 @@ package org.hatdex.hat.api.json
 import play.api.libs.json._
 
 import scala.util.Try
+import java.{ util => ju }
```

Comment:
This looks weird, why not `import java.utils.UUID`?

Reply:
Agreed, looks like one of the tools is trying to be smart and failing.

Reply:
Just for history sake, that was not my idea! :)